lib/engine/game/base.rb

```diff
@@ -305,6 +305,10 @@ def process_action(action)
         @actions << action
         next_round! while @round.finished? && !@finished
         self
+      rescue GameError => e
+        # Fill in the action id
+        e.action_id = current_action_id
+        raise
```

Comment:
should we raise game error?

Reply:
I thought "raise" bubbled the current exception up

Reply:
it does

Reply:
Toby, when you have a mo, could you rereview?

Reply:
so this doesn't in dev?

Reply:
do you have access to rack env?

Reply:
it's be in opal world, so no. but i could pass it in somehow.

Reply:
can you just pass it in as a need